### DISCUSSION TITLE
Running yarn opens debugger

### DISCUSSION THREAD

**Discussion by yesmeck** [community/community#26849] [Codespaces]
<p><img src="https://user-images.githubusercontent.com/465125/181100270-b5dc7b61-67a1-4454-8c37-4ad3d551a5f9.png" alt="Screen Shot 2020-09-24 at 11.58.42" data-base62-sha1="pCuxUL4QaYMwTGqtSTuqD08mfbG" width="683" height="160"></p>

**Comment by yesmeck** [comment]
<p>Actually, whatever node scripts I run opens the debugger, this is weird.</p>
<p><img src="https://user-images.githubusercontent.com/465125/181100272-5c5ef3ae-b7ab-489f-be64-68ad5238c268.png" alt="Screen Shot 2020-09-24 at 12.04.54" data-base62-sha1="xUUQ8Ukya8jKX64oIPavPDQXN9K" width="593" height="117"></p>

**Comment by maiertech** [comment, marked as answer]
<p>In your workspace settings or remote settings you can set <code>"debug.javascript.autoAttachFilter": "onlyWithFlag"</code> or to <code>off</code>.</p>

**Comment by Chuxel** [comment]
<p>This is <a href="https://github.com/microsoft/vscode/issues/106983" rel="noopener nofollow ugc">https://github.com/microsoft/vscode/issues/106983</a>. Auto-attach should not be defaulted “on”, but in the near term you can disable with the this setting.</p>
<aside class="quote no-group" data-username="454de6e" data-post="3" data-topic="133848" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 454de6e:</div>
<blockquote>
<p>In your workspace settings or remote settings you can set <code>"debug.javascript.autoAttachFilter": "onlyWithFlag"</code> or to <code>off</code> .</p>
</blockquote>
</aside>

**Comment by ianwalter** [comment]
<p>It’s really frustrating that this is on by default.</p>
